concordium-node/README.md

```diff
@@ -125,7 +127,7 @@ extra-include-dirs:
 - /opt/homebrew/include/
 ```
 
-### M1 MacOS Specific
+### Apple silicon specific
```

Comment:
This is less clear.

Reply:
It is however more correct. Would you rather have something like M1/M2 MacOS Specific?

Reply:
I would much rather keep it as documented in this PR. The generalised term for the individual processors are indeed Apple silicon, and it is just annoying to repeatedly add individual product names.

Right now my chip (M2) does not go in under this header however it should.

Furthermore the term M1 MacOS is simply invalid and does not exist. My MacOS is Ventura while my chip is an M2.

Reply:
The instructions are related to the operating system + CPU together. If I ran linux on M1 they would not apply.

Reply:
Okay based on how the statement is specified it sounds a lot like the MacOS is M1. 

_Apple silicon MacOS specific_ or _M1/M2 MacOS specific_? or would you rather we enter each individual processor in the title?